### DISCUSSION TITLE
16:9 cropping for media

### DISCUSSION THREAD

**Discussion by Freeplayg** [cheeaun/phanpy#23] [Ideas]
I get that this is mostly opinionated, but I do think it's probably best for media to be displayed in 16:9 as that's what the most common ratio is

**Comment by Freeplayg** [comment]
Actually that wasn't mentioned in the design decisions so that actually probably wasn't opinionated lol

**Comment by cheeaun** [comment]
Yeah I researched somewhere as well that 16:9 seems "common".

But from my observations:

- It's really not that "common" for images. Maybe yes for videos.
- No one's really posting images that's optimised for 16:9 cropping (too much work for user's end?)
- There's "focal setting" on Mastodon to fix the issue but I don't see anyone's using it (maybe too much work too?)
- The cropping ratio/dimensions should change based on other factors like dimension of the status component, viewport dimensions, and how many medias attached. E.g. if there are 4 photos posted, 16:9 ratio is no longer useful - and this is just one of the use-cases 😫

Overall, I don't see anything "common" or if there's a "best" answer for this (yet) 🙇‍♂️

Will continue observing and experiment 😉
